config/services.yml

```diff
@@ -44,12 +44,16 @@ services:
     factory: [ '@distributionapiclient.circuit_breaker.factory', 'create' ]
     arguments: [ '@distributionapiclient.circuit_breaker.settings' ]
 
+  distributionapiclient.shop_data_provider:
```

Comment:
Is this a new service? If yes, we should use FQCN instead.

Reply:
Done